flytekit/image_spec/image_spec.py

```diff
@@ -44,6 +44,7 @@ class ImageSpec:
         pip_index: Specify the custom pip index url
         registry_config: Specify the path to a JSON registry config file
         commands: Command to run during the building process
+        _is_force_push: forcefully pushed to the registry
```

Comment:
```suggestion
```

Reply:
It's used internally, so we don't need to add it here.

Reply:
updated